src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -77,6 +78,7 @@
       <_runtimePackagePath>$(PackagesDir)/$(_runtimePackageId.ToLowerInvariant())/$(_runtimePackageVersion)</_runtimePackagePath>
 
       <_crossGenPath>$(_runtimePackagePath)/tools$(_crossDir)/crossgen$(ApplicationFileExtension)</_crossGenPath>
+      <_crossGenPath Condition="'$(TargetArchitecture)' == 'arm' AND '$(OS)' == 'Windows_NT'">$(_runtimePackagePath)/tools$(_obsoleteCrossDir)/crossgen$(ApplicationFileExtension)</_crossGenPath>
```

Comment:
If this is `obsolete`, then why are we still using it?  If it is valid (i.e. not obsolete) on Windows, then we should call this property something else.

Reply:
This property is temporary to allow us to keep the `x86_arm` directory around long enough to move all the upstack dependents off of it and onto `x64_arm`.  Once there are no more dependents, this will be removed.  I suppose we could call it `deprecated`, but the transition period should be so brief that I'm not sure the terminology is important.

Reply:
Actually, sorry, I missed your emphasis on Windows.  You're right, since this is staying on Windows for now, `obsolete` is not a good choice.